feature-libs/product-configurator/rulebased/styles/mixins/_configurator-form-group.scss

```diff
@@ -3,27 +3,7 @@
     width: 100%;
     display: flex;
     flex-direction: row;
-
-    // TODO: this block should be remove after productConfiguratorAttributeTypesV2 feature flag has been removed
-    // TODO: start of the block
-    align-items: center;
-    @include media-breakpoint-up(md) {
```

Comment:
Out of curiosity: Why don't we need that with the new feature flag always active?

Reply:
as discussed